WeatherApp/Extension/String+Utils.swift

```diff
@@ -8,10 +8,15 @@
 import UIKit
 
 extension String {
+    
     func width(withConstrainedHeight height: CGFloat, font: UIFont) -> CGFloat {
```

Comment:
func textHeight(width: CGFloat, attributes: [NSAttributedString.Key: Any]?) -> CGFloat {
        let textStorage = NSTextStorage(string: self)
        let textContainer = NSTextContainer(size: CGSize(width: width, height: .infinity))
        let layoutManager = NSLayoutManager()
        layoutManager.addTextContainer(textContainer)
        textStorage.addLayoutManager(layoutManager)
        if let attributes = attributes {
            textStorage.addAttributes(attributes, range: NSMakeRange(0, textStorage.length))
        }
        textContainer.lineFragmentPadding = 0
        layoutManager.glyphRange(for: textContainer)
        return layoutManager.usedRect(for: textContainer).size.height
    }